src/main/electron-app/runnables/dock-visibility/hide-dock-for-last-closed-window.injectable.ts

```diff
@@ -20,7 +20,7 @@ const hideDockForLastClosedWindowInjectable = getInjectable({
       run: () => {
         const visibleWindows = pipeline(
           getLensWindows(),
-          filter(window => !!window.visible),
+          filter(window => !!window.isVisible),
         );
```

Comment:
This code has been duplicated in several places, it probably should be its own injectable.

Reply:
Sure. Normally we've done it when same thing is duplicated 3-4 times. But this is clearly case in which we don't have to wait since we know already it's not going to be wrong abstraction. :) Done.